### ISSUE TITLE
Review lab10 Enrico Capuano s317038

### ISSUE THREAD

**Issue by enricocapuano** [Francesco1102/Computational-Intelligence-23-24#5]
First of all, very good job on this code. You used in avery smart way the Monte Carlo method in the Tic Tac Toe game.
Also, I really appreciated the idea of checking the simmetry of the game. The code itself is very well written, but I would have appreciated to see some more comments to better understand the functioning.
It is nice that you decided to print the board at each move: in this way it is possible to check the moves that the players are doing.

In conclusion, you did a very good job, so well done!
I hope that my review will be appreciated by you.
